xilem/ARCHITECTURE.md

```diff
@@ -1,27 +1,56 @@
 # ARCHITECTURE
-Xilem is a framework that aims to provide a performant and productive option for Rust GUI.
 
-This document describes the high-level architecture of xilem. If you want to familiarize yourself with the code base, you are just in the right place!
+Xilem is a family of high-level GUI frameworks.
+Xilem apps are written with idiomatic Rust code, with little to no reliance on macros and DSLs.
```

Comment:
Fwiw, I think that Xilem view functions are pretty DSL-like, and I'm also not sure the Rust code you write is entirely idiomatic.

But idk, this is quite nitpicky. I don't really have a better suggestion.